packages/react-menu/e2e/support.js

```diff
@@ -0,0 +1,2 @@
+// workaround for https://github.com/cypress-io/cypress/issues/8599
+import '@fluentui/scripts/cypress/support';
```

Comment:
pls update also migration generator accordingly. this or follow-up PR works. thx